releasing/plugin-commands-deploy/test/deploy.test.ts

```diff
@@ -46,6 +59,10 @@ test('deploy', async () => {
     fs.writeFileSync(`${name}/index.js`, '', 'utf8')
   })
 
+  await writeYamlFile('pnpm-workspace.yaml', { packages: ['*'] })
+  crossSpawn.sync(pnpmBin, ['install', '--ignore-scripts', '--store-dir=../store', `--registry=http://localhost:${REGISTRY_MOCK_PORT}`])
+  fs.rmSync('pnpm-lock.yaml')
```

Comment:
A little bit of a hack for testing purposes... an assertion below checks that this file _does not exist_ (as a side effect of the `install` called internally to `deploy`) and wanted to maintain that assertion.